src/engraving/rw/read410/tread.cpp

```diff
@@ -2717,7 +2719,13 @@ bool TRead::readProperties(Chord* ch, XmlReader& e, ReadContext& ctx)
         TRead::read(cl, e, ctx);
         ch->add(cl);
     } else if (tag == "combineVoice") {
-        ch->setCombineVoice(e.readBool());
+        int val = e.readInt();
```

Comment:
Even though the "AutoOnOff" enum is unlikely to change, I think it's not ideal to write it to files as integers. [We have bad experiences with that.](https://github.com/musescore/MuseScore/issues/24060#issuecomment-2299665318) Instead, I'd write the value as a string, and add relevant methods to TConv. It's okay though to do that for 4.5 and not for 4.4.x.

Reply:
Ah thanks for bringing this up - we do in fact write strings for `AutoOnOff` so this was my mistake.